richeditor/src/main/assets/rich_editor.js

```diff
@@ -86,6 +86,10 @@ RE.setPlaceholder = function(placeholder) {
     RE.editor.setAttribute("placeholder", placeholder);
 }
 
+RE.setInputEnabled = function(inputEnabled) {
+    RE.editor.contentEditable = String(inputEnabled);
```

Comment:
Needs to be set as a string: https://developer.mozilla.org/en/docs/Web/API/HTMLElement/contentEditable